### PR TITLE
Move opentelemetry statsd example to examples repository

### PR DESCRIPTION
**Pull request by jack-berg**
Depends on first merging corresponding PR here: https://github.com/newrelic/newrelic-opentelemetry-examples/pull/621

### CODE REVIEW

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-intro.mdx [hunk 1]
```diff
@@ -8,6 +8,7 @@ metaDescription: Using the OpenTelemetry Collector for infrastructure monitoring
 freshnessValidatedDate: 2024-05-14
 redirect:
  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-redis
+ - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-statsd
```
Comment (ally-sassman) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-intro.mdx:11]:
```suggestion
 - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-statsd
 - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-statsd/
```

Comment (ally-sassman) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-intro.mdx:11]: Added the redirect link from the deleted page

**Review by ghost — DISMISSED** [review]

**Comment by ally-sassman** [comment]
@jack-berg I'll be on FTO from Wednesday to next Tuesday, so if you want this merged sooner, go ahead and ping #help-documentation and they'll take care of it for you.

**Review by ghost — DISMISSED** [review]

**Comment by jack-berg** [comment]
Superseded by #17742.

**Pull request closed without merging**